app/controllers/sign_up_sheet_controller.rb

```diff
@@ -421,7 +421,7 @@ def switch_original_topic_to_approved_suggested_topic
     assignment = AssignmentParticipant.find(params[:id]).assignment
     team_id = TeamsUser.team_id(assignment.id, session[:user].id)
     original_topic_id = SignedUpTeam.topic_id(assignment.id.to_i, session[:user].id)
-    SignUpTopic.find(params[:topic_id]).update_attribute(:private_to, nil) if SignUpTopic.exists?(params[:topic_id])
+    SignUpTopic.find_by(topic_id: params[:topic_id]).update_attribute(:private_to, nil) if SignUpTopic.exists?(topic_id: params[:topic_id])
```

Comment:
Are you sure topic_id is the right field to search by? Usually find() defaults to searching by that class' ID, and other files such as assessment360_controller.rb on line 131 have the find_by setup as:
![image](https://user-images.githubusercontent.com/41524135/159187825-f5011c20-a047-49a8-9059-2c818d5b6e81.png)